apps/jobs/models.py

```diff
@@ -36,8 +40,10 @@ class Submission(TimeStampedModel):
         (FINISHED, FINISHED),
     )
 
-    participant_team = models.ForeignKey(ParticipantTeam, related_name='submissions')
-    challenge_phase = models.ForeignKey(ChallengePhase, related_name='submissions')
+    participant_team = models.ForeignKey(
```

Comment:
Cool I will add another field in the model named `submitted_by` to store the information of the user and that will not affect the business logic. I will just need to store an additional field.